src/main/java/net/mcreator/ui/modgui/LootTableGUI.java

```diff
@@ -123,6 +141,18 @@ public LootTableGUI(MCreator mcreator, ModElement modElement, boolean editingMod
 						type.setSelectedItem("Block");
 					} else if (currName.startsWith("entities/")) {
 						type.setSelectedItem("Entity");
+					} else if (currName.startsWith("archaeology/")) {

```

Comment:
This code is only meant for custom MEs for block and entity types where one select such entry from dropdown.

Thus those are not relavant. Please remove those checks